.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,72 @@
+name: Deploy
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@master
+
+      - name: start deployment
+        uses: bobheadxi/deployments@master
```

Comment:
Interesting -- how did you find https://github.com/bobheadxi/deployments, out of curiosity?

Reply:
I searched the Actions marketplace. I've no prior experience of this, but it seemed to do the trick. I picked this one as it seemed to focus on manipulating the deployments API rather than doing the deployment.

Reply:
Gotcha, that makes sense! I only found out a few days ago that Actions is so new that GitHub is sponsoring a three-week remote hackathon for it, btw -- https://githubhackathon.com/. :)

/doesthismeanwe'remodern

Reply:
They've been around in beta for a while, but they definitely got better with the roll-out.